crates/sui-grpc-rosetta/src/account.rs

```diff
@@ -14,16 +19,16 @@ use tracing::info;
 use crate::errors::Error;
 use crate::types::{
     AccountBalanceRequest, AccountBalanceResponse, AccountCoinsRequest, AccountCoinsResponse,
-    Amount, Coin, Currencies, Currency, SubAccountType, SubBalance,
+    Amount, Coin, CoinID, CoinIdentifier, Currencies, Currency, SubAccountType, SubBalance,
 };
 use crate::{OnlineServerContext, SuiEnv};
 use std::time::Duration;
-use sui_sdk::error::SuiRpcResult;
+use sui_types::base_types::{ObjectID, SequenceNumber};
 use sui_types::messages_checkpoint::CheckpointSequenceNumber;
 
 /// Get an array of all AccountBalances for an AccountIdentifier and the BlockIdentifier
 /// at which the balance lookup was performed.
-/// [Rosetta API Spec](https://www.rosetta-api.org/docs/AccountApi.html#accountbalance)
+/// [Rosetta API Spec](https://docs.cdp.coinbase.com/api-reference/mesh/account/get-an-account-balance)
```

Comment:
thank you for updating the apis, i guess we may need to rename rosetta to...mesh? 🤢